packages/simulator/simulator/passengers.js

```diff
@@ -89,29 +89,46 @@ const generatePassengers = (kommuner) =>
     toArray()
   )
 const createPassengerFromAddress = ({ home, work }) => {
-  console.log("createPassengerFromAddress")
-  const residence = { name: `${home.name}, ${home.localadmin}`, ...home.position}
-  const workplace = { name: `${work.name}, ${work.localadmin}`, ...work.position}
+  console.log('createPassengerFromAddress')
+  const residence = {
+    name: `${home.name}, ${home.localadmin}`,
+    ...home.position,
+  }
+  const workplace = {
+    name: `${work.name}, ${work.localadmin}`,
+    ...work.position,
+  }
 
   const offset = virtualTime.offset // Sim starts at an offset from midnight, so we need to subctract that offset from the time
-  const fiveAm  = (5 * 60 * 60) - offset
-  const tenAm   = (10 * 60 * 60) - offset
-  const threePm = (15 * 60 * 60) - offset
-  const eightPm = (20 * 60 * 60) - offset
+  const fiveAm = 5 * 60 * 60 - offset
+  const tenAm = 10 * 60 * 60 - offset
+  const threePm = 15 * 60 * 60 - offset
+  const eightPm = 20 * 60 * 60 - offset
   const name = names[Math.floor(Math.random() * names.length)]
 
   return new Passenger({
     journeys: [
-      { id: safeId(), pickup: residence, destination: workplace, timeWindow: [[fiveAm, tenAm]], status: 'Väntar' },
-      { id: safeId(), pickup: workplace, destination: residence, timeWindow: [[threePm, eightPm]], status: 'Väntar' },
+      {
+        id: safeId(),
```

Comment:
safeId ska inte vara kvar här eller på rad 108/126 nedan eftersom det är flyttat till modellernas konstruktorer i main, du kan behöva rebase:a på main eller merga in main.

Samma sak med console.log:en för att skapa en passagerare ovan.